packages/dcos-integration-test/extra/common.py

```diff
@@ -0,0 +1,3 @@
+import pytest
+
+xfailflake = pytest.mark.xfail(strict=False)
```

Comment:
Let's have a comment here with a description of why we have this and when to add / remove this marker.

Reply:
Can do this in a follow-up though